locales/en.json

```diff
@@ -39,19 +39,18 @@
   "https://www.canada.ca/en/revenue-agency/services/e-services/e-services-individuals/account-individuals.html?utm_campaign=not-applicable&utm_medium=vanity-url&utm_source=canada-ca_my-cra-account": "https://www.canada.ca/en/revenue-agency/services/e-services/e-services-individuals/account-individuals.html?utm_campaign=not-applicable&utm_medium=vanity-url&utm_source=canada-ca_my-cra-account",
   "canada.ca/taxes-help": "canada.ca/taxes-help",
   "canada.ca/my-cra-account": "canada.ca/my-cra-account",
-  "service.name": "Claim Tax Benefits",
+  "service.name": "Claim benefits by filing taxes",
```

Comment:
Wait, this changes the text in a bunch of places in the app (check the next page for example). Is this the intention?